uhdm-plugin/uhdmsurelogastfrontend.cc

```diff
@@ -87,100 +89,12 @@ unsigned int executeCompilation(
 	if (noFErrors == false) {
 		noFatalErrors = false;
 	}
-
-	std::string ext_command = clp->getExeCommand();
-	if (!ext_command.empty()) {
-		std::string directory = symbolTable->getSymbol(clp->getFullCompileDir());
-		std::string fileList = directory + "/file.lst";
-		std::string command = ext_command + " " + fileList;
-		int result = system(command.c_str());
-		codedReturn |= result;
-		std::cout << "Command result: " << result << std::endl;
-	}
-	clp->logFooter();
-	if (diff_comp_mode && fileunit) {
-		SURELOG::Report* report = new SURELOG::Report();
-		std::pair<bool, bool> results =
-				report->makeDiffCompUnitReport(clp, symbolTable);
-		success = results.first;
-		noFatalErrors = results.second;
-		delete report;
-	}
-	clp->cleanCache();	// only if -nocache
-	delete clp;
-	delete symbolTable;
-	delete errors;
+	//SURELOG::shutdown_compiler(compiler);
```

Comment:
Why is this commented out instead of deleted?

Reply:
It was leftover from the tests I did.
I refactored ``executeCompilation`` function and now I delete them after yosys preprocessing.